### PR TITLE
HDDS-1093. Configuration tab in OM/SCM ui is not displaying the correct values

### PR DESCRIPTION
**Pull request by vivekratnavel**
Configuration tab in OM/SCM ui is not displaying the correct/configured values, rather it is displaying the default values.

### CODE REVIEW

**Comment by vivekratnavel** [comment]
@bharatviswa504 @avijayanhwx @arp7 Please review when you find time

File: hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/TestOzoneConfiguration.java [hunk 1]
```diff
@@ -0,0 +1,171 @@
...
+      appendPropertyByTag("dfs.random.key", "XYZ", "MYCUSTOMTAG");
+      endConfig();
+
+      Path fileResource = new Path(CONFIG);
+      conf.addResource(fileResource);
+      assertEq(conf.getAllPropertiesByTag("MYCUSTOMTAG").getProperty("dfs.random.key"), "XYZ");
+    } finally {
+      out.close();
+    }
+    try {
```
Comment (swagle) [inline hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/TestOzoneConfiguration.java]: Better to use try construct that closes the Closeable vs explicit close. syntactic sugar

File: hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/TestOzoneConfiguration.java [hunk 2]
```diff
@@ -0,0 +1,171 @@
...
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class TestOzoneConfiguration {
+
+  private Configuration conf;
+  final static String CONFIG = new File("./test-config-TestConfiguration.xml").getAbsolutePath();
```
Comment (swagle) [inline hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/TestOzoneConfiguration.java]: Better to use junit's TemporaryFolder and let framework take care of the cleanup as well as create etc.

File: hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/TestOzoneConfiguration.java [hunk 3]
```diff
@@ -0,0 +1,171 @@
...
+  final static String CONFIG_CORE = new File("./core-site.xml").getAbsolutePath();
+
+  private BufferedWriter out;
```
Comment (swagle) [inline hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/TestOzoneConfiguration.java]: In general not too excited about sharing streams as a coding best practice, others reviewers can chime in but better to localize this vs deal with side effect code in tearDown.

**Comment by brownscott** [comment]
-- 
Sent from my Android phone with GMX Mail. Please excuse my brevity.On 2019-02-27, 9:55 p.m. Siddharth <notifications@github.com> wrote:
@swagle commented on this pull request.



In hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/TestOzoneConfiguration.java:
> +import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+
+import java.io.BufferedWriter;
+import java.io.File;
+import java.io.FileWriter;
+import java.io.IOException;
+
+public class TestOzoneConfiguration {
+
+  private Configuration conf;
+  final static String CONFIG = new File("./test-config-TestConfiguration.xml").getAbsolutePath();
+  final static String CONFIG_CORE = new File("./core-site.xml").getAbsolutePath();
+
+  private BufferedWriter out;

In general not too excited about sharing streams as a coding best practice, others reviewers can chime in but better to localize this vs deal with side effect code in tearDown.

—You are receiving this because you are subscribed to this thread.Reply to this email directly, view it on GitHub, or mute the thread.
{"api_version":"1.0","publisher":{"api_key":"05dde50f1d1a384dd78767c55493e4bb","name":"GitHub"},"entity":{"external_key":"github/apache/hadoop","title":"apache/hadoop","subtitle":"GitHub repository","main_image_url":"https://github.githubassets.com/images/email/message_cards/header.png","avatar_image_url":"https://github.githubassets.com/images/email/message_cards/avatar.png","action":{"name":"Open in GitHub","url":"https://github.com/apache/hadoop"}},"updates":{"snippets":[{"icon":"PERSON","message":"@swagle commented on #527"}],"action":{"name":"View Pull Request","url":"https://github.com/apache/hadoop/pull/527#pullrequestreview-208858889"}}}
[
{
"@context": "http://schema.org",
"@type": "EmailMessage",
"potentialAction": {
"@type": "ViewAction",
"target": "https://github.com/apache/hadoop/pull/527#pullrequestreview-208858889",
"url": "https://github.com/apache/hadoop/pull/527#pullrequestreview-208858889",
"name": "View Pull Request"
},
"description": "View this Pull Request on GitHub",
"publisher": {
"@type": "Organization",
"name": "GitHub",
"url": "https://github.com"
}
}
]

File: hadoop-hdds/framework/src/main/resources/webapps/static/ozone.js [hunk 4]
```diff
@@ -308,7 +308,6 @@
             ctrl.convertToArray(response.data);
             ctrl.configs = Object.values(ctrl.keyTagMap);
             ctrl.component = 'All';
-            console.log("ajay -> " + JSON.stringify(ctrl.configs));
```
Comment (avijayanhwx) [inline hadoop-hdds/framework/src/main/resources/webapps/static/ozone.js]: I believe this file needs to be removed.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 39 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 2 new or modified test files. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 11 | Maven dependency ordering for branch |
| +1 | mvninstall | 1042 | trunk passed |
| +1 | compile | 72 | trunk passed |
| +1 | checkstyle | 30 | trunk passed |
| +1 | mvnsite | 65 | trunk passed |
| +1 | shadedclient | 777 | branch has no errors when building and testing our client artifacts. |
| +1 | findbugs | 97 | trunk passed |
| +1 | javadoc | 57 | trunk passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 12 | Maven dependency ordering for patch |
| +1 | mvninstall | 62 | the patch passed |
| -1 | jshint | 75 | The patch generated 294 new + 1942 unchanged - 1053 fixed = 2236 total (was 2995) |
| +1 | compile | 66 | the patch passed |
| +1 | javac | 66 | the patch passed |
| -0 | checkstyle | 23 | hadoop-hdds: The patch generated 12 new + 0 unchanged - 0 fixed = 12 total (was 0) |
| +1 | mvnsite | 54 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 766 | patch has no errors when building and testing our client artifacts. |
| +1 | findbugs | 109 | the patch passed |
| +1 | javadoc | 53 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 87 | common in the patch failed. |
| +1 | unit | 32 | framework in the patch passed. |
| +1 | asflicense | 26 | The patch does not generate ASF License warnings. |
| | | 3616 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hdds.security.x509.certificate.client.TestDefaultCertificateClient |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-527/1/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/527 |
| Optional Tests |  dupname  asflicense  compile  javac  javadoc  mvninstall  mvnsite  unit  shadedclient  findbugs  checkstyle  jshint  |
| uname | Linux 37fa4eb47056 3.13.0-153-generic #203-Ubuntu SMP Thu Jun 14 08:52:28 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / cbf82fa |
| maven | version: Apache Maven 3.3.9 |
| Default Java | 1.8.0_191 |
| findbugs | v3.1.0-RC1 |
| jshint | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/1/artifact/out/diff-patch-jshint.txt |
| checkstyle | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/1/artifact/out/diff-checkstyle-hadoop-hdds.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/1/artifact/out/patch-unit-hadoop-hdds_common.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/1/testReport/ |
| Max. process+thread count | 304 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/common hadoop-hdds/framework U: hadoop-hdds |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/1/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/conf/OzoneConfiguration.java [hunk 5]
```diff
@@ -161,4 +163,31 @@ public static void activate() {
     Configuration.addDefaultResource("ozone-default.xml");
     Configuration.addDefaultResource("ozone-site.xml");
   }
+
+  /**
+   * The super class method getAllPropertiesByTag
```
Comment (elek) [inline hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/conf/OzoneConfiguration.java]: Do we need an other (HADOOP) jira to fix it on the hadoop 3.3 line, too?

**Comment by elek** [comment]
I am just wondering if it's the same issue as https://issues.apache.org/jira/browse/HDDS-611

**Comment by vivekratnavel** [comment]
> I am just wondering if it's the same issue as https://issues.apache.org/jira/browse/HDDS-611

Thanks @elek ! Marked it a duplicate.

File: hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/conf/OzoneConfiguration.java (hunk 5 above)
Comment (vivekratnavel) [reply hadoop-hdds/common/src/main/java/org/apache/hadoop/hdds/conf/OzoneConfiguration.java]: Yes @elek

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 42 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 2 new or modified test files. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 26 | Maven dependency ordering for branch |
| +1 | mvninstall | 1075 | trunk passed |
| +1 | compile | 75 | trunk passed |
| +1 | checkstyle | 30 | trunk passed |
| +1 | mvnsite | 71 | trunk passed |
| +1 | shadedclient | 786 | branch has no errors when building and testing our client artifacts. |
| +1 | findbugs | 100 | trunk passed |
| +1 | javadoc | 60 | trunk passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 11 | Maven dependency ordering for patch |
| +1 | mvninstall | 66 | the patch passed |
| -1 | jshint | 76 | The patch generated 294 new + 1942 unchanged - 1053 fixed = 2236 total (was 2995) |
| +1 | compile | 65 | the patch passed |
| +1 | javac | 65 | the patch passed |
| -0 | checkstyle | 24 | hadoop-hdds: The patch generated 5 new + 0 unchanged - 0 fixed = 5 total (was 0) |
| +1 | mvnsite | 54 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 772 | patch has no errors when building and testing our client artifacts. |
| +1 | findbugs | 115 | the patch passed |
| +1 | javadoc | 56 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 93 | common in the patch failed. |
| +1 | unit | 33 | framework in the patch passed. |
| +1 | asflicense | 27 | The patch does not generate ASF License warnings. |
| | | 3713 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hdds.security.x509.certificate.client.TestDefaultCertificateClient |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-527/2/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/527 |
| Optional Tests |  dupname  asflicense  compile  javac  javadoc  mvninstall  mvnsite  unit  shadedclient  findbugs  checkstyle  jshint  |
| uname | Linux c15708c89542 3.13.0-153-generic #203-Ubuntu SMP Thu Jun 14 08:52:28 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / eae3db9 |
| maven | version: Apache Maven 3.3.9 |
| Default Java | 1.8.0_191 |
| findbugs | v3.1.0-RC1 |
| jshint | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/2/artifact/out/diff-patch-jshint.txt |
| checkstyle | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/2/artifact/out/diff-checkstyle-hadoop-hdds.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/2/artifact/out/patch-unit-hadoop-hdds_common.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/2/testReport/ |
| Max. process+thread count | 306 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/common hadoop-hdds/framework U: hadoop-hdds |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/2/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 54 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 2 new or modified test files. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 15 | Maven dependency ordering for branch |
| +1 | mvninstall | 1212 | trunk passed |
| +1 | compile | 89 | trunk passed |
| +1 | checkstyle | 35 | trunk passed |
| +1 | mvnsite | 75 | trunk passed |
| +1 | shadedclient | 802 | branch has no errors when building and testing our client artifacts. |
| +1 | findbugs | 100 | trunk passed |
| +1 | javadoc | 57 | trunk passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 11 | Maven dependency ordering for patch |
| +1 | mvninstall | 66 | the patch passed |
| -1 | jshint | 83 | The patch generated 294 new + 1942 unchanged - 1053 fixed = 2236 total (was 2995) |
| +1 | compile | 69 | the patch passed |
| +1 | javac | 69 | the patch passed |
| +1 | checkstyle | 24 | the patch passed |
| +1 | mvnsite | 54 | the patch passed |
| +1 | whitespace | 0 | The patch has no whitespace issues. |
| +1 | shadedclient | 759 | patch has no errors when building and testing our client artifacts. |
| +1 | findbugs | 110 | the patch passed |
| +1 | javadoc | 53 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 87 | common in the patch failed. |
| +1 | unit | 31 | framework in the patch passed. |
| +1 | asflicense | 26 | The patch does not generate ASF License warnings. |
| | | 3867 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hdds.security.x509.certificate.client.TestDefaultCertificateClient |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-527/3/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/527 |
| Optional Tests |  dupname  asflicense  compile  javac  javadoc  mvninstall  mvnsite  unit  shadedclient  findbugs  checkstyle  jshint  |
| uname | Linux 7544400e110b 3.13.0-153-generic #203-Ubuntu SMP Thu Jun 14 08:52:28 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 8b72aea |
| maven | version: Apache Maven 3.3.9 |
| Default Java | 1.8.0_191 |
| findbugs | v3.1.0-RC1 |
| jshint | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/3/artifact/out/diff-patch-jshint.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/3/artifact/out/patch-unit-hadoop-hdds_common.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/3/testReport/ |
| Max. process+thread count | 307 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/common hadoop-hdds/framework U: hadoop-hdds |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/3/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

File: hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/TestOzoneConfiguration.java [hunk 6]
```diff
@@ -0,0 +1,145 @@
...
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+package org.apache.hadoop.hdds.conf;
+
+import org.apache.hadoop.conf.Configuration;
+import org.apache.hadoop.fs.Path;
+import org.junit.*;
```
Comment (bharatviswa504) [inline hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/TestOzoneConfiguration.java]: Minor Nit: Can we avoid using *

File: hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/TestOzoneConfiguration.java [hunk 7]
```diff
@@ -0,0 +1,145 @@
...
+    if (isFinal) {
+      out.write("<final>true</final>");
+    }
+    out.write("<tag>");
+    out.write(tag);
+    out.write("</tag>");
+    out.write("</property>\n");
+  }
+
+  private static void assertEq(Object a, Object b) {
```
Comment (bharatviswa504) [inline hadoop-hdds/common/src/test/java/org/apache/hadoop/hdds/conf/TestOzoneConfiguration.java]: Instead we can directly use assertEquals, we don't need to do System.out.println

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 20 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 2 new or modified test files. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 27 | Maven dependency ordering for branch |
| +1 | mvninstall | 1189 | trunk passed |
| +1 | compile | 77 | trunk passed |
| +1 | checkstyle | 29 | trunk passed |
| +1 | mvnsite | 67 | trunk passed |
| +1 | shadedclient | 777 | branch has no errors when building and testing our client artifacts. |
| +1 | findbugs | 105 | trunk passed |
| +1 | javadoc | 58 | trunk passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 12 | Maven dependency ordering for patch |
| +1 | mvninstall | 72 | the patch passed |
| -1 | jshint | 294 | The patch generated 2954 new + 0 unchanged - 0 fixed = 2954 total (was 0) |
| +1 | compile | 69 | the patch passed |
| +1 | javac | 69 | the patch passed |
| +1 | checkstyle | 25 | the patch passed |
| +1 | mvnsite | 57 | the patch passed |
| -1 | whitespace | 0 | The patch has 75 line(s) that end in whitespace. Use git apply --whitespace=fix <<patch_file>>. Refer https://git-scm.com/docs/git-apply |
| -1 | whitespace | 1 | The patch 19849  line(s) with tabs. |
| +1 | shadedclient | 955 | patch has no errors when building and testing our client artifacts. |
| +1 | findbugs | 117 | the patch passed |
| +1 | javadoc | 55 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 76 | common in the patch failed. |
| +1 | unit | 33 | framework in the patch passed. |
| +1 | asflicense | 26 | The patch does not generate ASF License warnings. |
| | | 4240 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hdds.security.x509.certificate.client.TestDefaultCertificateClient |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-527/4/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/527 |
| Optional Tests |  dupname  asflicense  compile  javac  javadoc  mvninstall  mvnsite  unit  shadedclient  findbugs  checkstyle  jshint  |
| uname | Linux 60f7c2a056db 4.4.0-139-generic #165~14.04.1-Ubuntu SMP Wed Oct 31 10:55:11 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 945b504 |
| maven | version: Apache Maven 3.3.9 |
| Default Java | 1.8.0_191 |
| findbugs | v3.1.0-RC1 |
| jshint | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/4/artifact/out/diff-patch-jshint.txt |
| whitespace | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/4/artifact/out/whitespace-eol.txt |
| whitespace | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/4/artifact/out/whitespace-tabs.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/4/artifact/out/patch-unit-hadoop-hdds_common.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/4/testReport/ |
| Max. process+thread count | 341 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/common hadoop-hdds/framework U: hadoop-hdds |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/4/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by bharatviswa504** [comment]
LGTM.
Could you confirm, whether these acceptance test failures are not related to this patch?

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 27 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 2 new or modified test files. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 29 | Maven dependency ordering for branch |
| +1 | mvninstall | 1198 | trunk passed |
| +1 | compile | 75 | trunk passed |
| +1 | checkstyle | 27 | trunk passed |
| +1 | mvnsite | 71 | trunk passed |
| +1 | shadedclient | 745 | branch has no errors when building and testing our client artifacts. |
| +1 | findbugs | 107 | trunk passed |
| +1 | javadoc | 57 | trunk passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 12 | Maven dependency ordering for patch |
| +1 | mvninstall | 70 | the patch passed |
| +1 | jshint | 385 | There were no new jshint issues. |
| +1 | compile | 73 | the patch passed |
| +1 | javac | 73 | the patch passed |
| +1 | checkstyle | 28 | the patch passed |
| +1 | mvnsite | 61 | the patch passed |
| -1 | whitespace | 0 | The patch has 75 line(s) that end in whitespace. Use git apply --whitespace=fix <<patch_file>>. Refer https://git-scm.com/docs/git-apply |
| -1 | whitespace | 1 | The patch 19849  line(s) with tabs. |
| +1 | shadedclient | 888 | patch has no errors when building and testing our client artifacts. |
| +1 | findbugs | 112 | the patch passed |
| +1 | javadoc | 53 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 74 | common in the patch failed. |
| +1 | unit | 32 | framework in the patch passed. |
| +1 | asflicense | 28 | The patch does not generate ASF License warnings. |
| | | 4228 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.hdds.security.x509.certificate.client.TestDefaultCertificateClient |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-527/5/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/527 |
| Optional Tests |  dupname  asflicense  compile  javac  javadoc  mvninstall  mvnsite  unit  shadedclient  findbugs  checkstyle  jshint  |
| uname | Linux 4127a35af4d8 4.4.0-139-generic #165-Ubuntu SMP Wed Oct 24 10:58:50 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 2c3ec37 |
| maven | version: Apache Maven 3.3.9 |
| Default Java | 1.8.0_191 |
| findbugs | v3.1.0-RC1 |
| whitespace | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/5/artifact/out/whitespace-eol.txt |
| whitespace | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/5/artifact/out/whitespace-tabs.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/5/artifact/out/patch-unit-hadoop-hdds_common.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/5/testReport/ |
| Max. process+thread count | 446 (vs. ulimit of 5500) |
| modules | C: hadoop-hdds/common hadoop-hdds/framework U: hadoop-hdds |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/5/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Comment by bharatviswa504** [comment]
Thank You @vivekratnavel  for the fix.
WIll commit this shortly.

**Comment by hadoop-yetus** [comment]
:broken_heart: **-1 overall**






| Vote | Subsystem | Runtime | Comment |
|:----:|----------:|--------:|:--------|
| 0 | reexec | 89 | Docker mode activated. |
||| _ Prechecks _ |
| +1 | @author | 0 | The patch does not contain any @author tags. |
| +1 | test4tests | 0 | The patch appears to include 3 new or modified test files. |
||| _ trunk Compile Tests _ |
| 0 | mvndep | 90 | Maven dependency ordering for branch |
| +1 | mvninstall | 1211 | trunk passed |
| +1 | compile | 1048 | trunk passed |
| +1 | checkstyle | 221 | trunk passed |
| +1 | mvnsite | 440 | trunk passed |
| +1 | shadedclient | 738 | branch has no errors when building and testing our client artifacts. |
| 0 | findbugs | 0 | Skipped patched modules with no Java source: hadoop-ozone/dist |
| +1 | findbugs | 291 | trunk passed |
| +1 | javadoc | 234 | trunk passed |
||| _ Patch Compile Tests _ |
| 0 | mvndep | 25 | Maven dependency ordering for patch |
| -1 | mvninstall | 20 | dist in the patch failed. |
| -1 | jshint | 195 | The patch generated 3087 new + 0 unchanged - 0 fixed = 3087 total (was 0) |
| +1 | compile | 957 | the patch passed |
| +1 | javac | 957 | the patch passed |
| +1 | checkstyle | 220 | the patch passed |
| +1 | mvnsite | 274 | the patch passed |
| +1 | shellcheck | 0 | There were no new shellcheck issues. |
| +1 | shelldocs | 30 | There were no new shelldocs issues. |
| +1 | whitespace | 1 | The patch has no whitespace issues. |
| +1 | shadedclient | 694 | patch has no errors when building and testing our client artifacts. |
| 0 | findbugs | 0 | Skipped patched modules with no Java source: hadoop-ozone/dist |
| +1 | findbugs | 323 | the patch passed |
| +1 | javadoc | 231 | the patch passed |
||| _ Other Tests _ |
| -1 | unit | 547 | hadoop-common in the patch failed. |
| +1 | unit | 113 | common in the patch passed. |
| +1 | unit | 49 | framework in the patch passed. |
| +1 | unit | 143 | server-scm in the patch passed. |
| +1 | unit | 35 | dist in the patch passed. |
| +1 | asflicense | 52 | The patch does not generate ASF License warnings. |
| | | 8812 | |


| Reason | Tests |
|-------:|:------|
| Failed junit tests | hadoop.util.TestReadWriteDiskValidator |
|   | hadoop.util.TestBasicDiskValidator |
|   | hadoop.util.TestDiskCheckerWithDiskIo |


| Subsystem | Report/Notes |
|----------:|:-------------|
| Docker | Client=17.05.0-ce Server=17.05.0-ce base: https://builds.apache.org/job/hadoop-multibranch/job/PR-527/6/artifact/out/Dockerfile |
| GITHUB PR | https://github.com/apache/hadoop/pull/527 |
| Optional Tests |  dupname  asflicense  compile  javac  javadoc  mvninstall  mvnsite  unit  shadedclient  findbugs  checkstyle  jshint  shellcheck  shelldocs  |
| uname | Linux e5b0179c4a92 3.13.0-153-generic #203-Ubuntu SMP Thu Jun 14 08:52:28 UTC 2018 x86_64 x86_64 x86_64 GNU/Linux |
| Build tool | maven |
| Personality | personality/hadoop.sh |
| git revision | trunk / 618e009 |
| maven | version: Apache Maven 3.3.9 |
| Default Java | 1.8.0_191 |
| shellcheck | v0.4.6 |
| findbugs | v3.1.0-RC1 |
| mvninstall | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/6/artifact/out/patch-mvninstall-hadoop-ozone_dist.txt |
| jshint | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/6/artifact/out/diff-patch-jshint.txt |
| unit | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/6/artifact/out/patch-unit-hadoop-common-project_hadoop-common.txt |
|  Test Results | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/6/testReport/ |
| Max. process+thread count | 1396 (vs. ulimit of 5500) |
| modules | C: hadoop-common-project/hadoop-common hadoop-hdds/common hadoop-hdds/framework hadoop-hdds/server-scm hadoop-ozone/dist U: . |
| Console output | https://builds.apache.org/job/hadoop-multibranch/job/PR-527/6/console |
| Powered by | Apache Yetus 0.9.0 http://yetus.apache.org |


This message was automatically generated.

**Pull request merged**